kayak_core/src/fragment.rs

```diff
@@ -2,12 +2,26 @@ use crate::{
     context_ref::KayakContextRef, styles::Style, Children, Index, OnEvent, Widget, WidgetProps,
 };
 
+/// Props used by the [`Fragment`] widget
 #[derive(Default, Debug, PartialEq, Clone)]
 pub struct FragmentProps {
     pub styles: Option<Style>,
     pub children: Option<crate::Children>,
 }
 
+/// The base widget, used to actually build and render children
+///
+/// # Props
+///
+/// __Type:__ [`FragmentProps`]
+///
+/// | Common Prop | Accepted |
+/// | :---------: | :------: |
+/// | `children`  | ✅        |
+/// | `styles`    | ✅        |
+/// | `on_event`  | ❌        |
+/// | `focusable` | ❌        |
+///
```

Comment:
Thoughts on this layout? For built-in widgets, I think it's nice to show right on the widget what common props it accepts. Although, I'm open to alternatives (as well as just not showing that information).